code/modules/mining/equipment/kinetic_crusher.dm

```diff
@@ -193,12 +193,14 @@
 		if(hammer_synced)
 			for(var/t in hammer_synced.trophies)
 				var/obj/item/crusher_trophy/T = t
+				if(istype(t,/obj/item/crusher_trophy/syndicate_trophy))
+					L.mob_size = MOB_SIZE_LARGE
 				T.on_mark_application(target, CM, had_effect)
 	var/target_turf = get_turf(target)
 	if(ismineralturf(target_turf))
 		var/turf/closed/mineral/M = target_turf
 		new /obj/effect/temp_visual/kinetic_blast(M)
-		M.gets_drilled(firer)
+		M.gets_drilled(firer,TRUE)
```

Comment:
this needs to use a named argument for the TRUE, so you can tell which argument of the proc is actually having TRUE passed to

Reply:
this language hurts my brain

Reply:
wait but the PKA doesnt need that it just uses
M.gets_drilled(firer, TRUE)

Reply:
the pka does it wrong